app/javascript/App/index.js

```diff
@@ -79,3 +82,4 @@ document.addEventListener('DOMContentLoaded', () => {
 })
 
 window.addEventListener('resize', debounce(setScreenSize), 400)
+window.addEventListener('resize', debounce(scrollHelper), 400)
```

Comment:
Dumb question - why is the scrollHelper used both app-wide here, as well as specifically in the NewsList?

Reply:
it's a race condition. the app-wide component will mount first. it checks for existence of the html before applying the fix. if the app-wide check happens before the NewsList mounts then the helper won't work. 

we could probably get away with invoking the helper in just the CoinDrawer as that is a child of the NewsList. but i don't think that optimization will buy us noticeable gains in performance. on a slower machine the mounting can happen slowly. it's probably safer way to make sure the fix works most of the time.